src/components/MediaMaker/Settings/VideoSettings.tsx

```diff
@@ -26,41 +25,23 @@ import ImageIcon from '@/icons/photo.svg';
 import TextIconSuccess from '@/icons/text-success.svg';
 import TextIcon from '@/icons/text.svg';
 import layoutStyle from '@/pages/index.module.scss';
-import { resetToDefaults, updateSettings } from '@/redux/slices/mediaMaker';
+import { resetToDefaults } from '@/redux/slices/mediaMaker';
 import MediaSettings, { ChangedSettings } from '@/types/Media/MediaSettings';
 import QueryParam from '@/types/QueryParam';
 import Reciter from '@/types/Reciter';
-import { logButtonClick, logEvent, logValueChange } from '@/utils/eventLogger';
+import { logButtonClick, logEvent } from '@/utils/eventLogger';
 
 type Props = {
   chaptersList: any[];
   reciters: Reciter[];
   seekToBeginning: () => void;
+  onSettingsUpdate: (settings: ChangedSettings, key?: keyof MediaSettings, value?: any) => void;
   getCurrentFrame: () => void;
   isFetching: boolean;
   inputProps: any;
   mediaSettings: MediaSettings;
 };
 
-const MEDIA_SETTINGS_TO_QUERY_PARAM = {
```

Comment:
Is there a reason why this was moved to `src/pages/quran-media-maker/index.tsx` as that file has so many lines of code already.